src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+
+import java.time.format.DateTimeFormatter;
+
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    private static int taskCount = 0;
+    private static final String TICK = "\u2713";
```

Comment:
Perhaps have the constants be `public`? Encapsulation is for variables: https://se-education.org/guides/conventions/java/index.html#variables

> Class variables should never be declared public unless the class is a data class with no behavior. This rule does not apply to constants.

So the getters are not needed for constants.